resolver/src/p2p/peer/fetcher.rs

```diff
@@ -33,17 +32,17 @@ enum SendError<S: Sender> {
 /// - Pending: Not successfully sent to a peer. Waiting to be retried by timeout.
 ///
 /// Both types of requests will be retried after a timeout if not resolved (i.e. a response or a
-/// cancelation). Upon retry, requests may either be placed in active or pending state again.
```

Comment:
most annoying thing is the American way is to use one `L` I guess for like every version of "cancel" (e.g. canceled, canceling) except I guess for "cancellation" where double-L is standard for both American and British

Reply:
english do b like that sometimes